api/collections/views.py

```diff
@@ -0,0 +1,382 @@
+        return node
+
+
+class CollectionList(generics.ListCreateAPIView, ODMFilterMixin):
+    """Organizer Collections organize projects and components. *Writeable*.
+
+    Paginated list of Project Organizer Collections ordered by their `date_modified`.
+    Each resource contains the full representation of the project organizer collection, meaning additional
+    requests to an individual Organizer Collection's detail view are not necessary.
+
+    The Project Organizer is a tool to allow the user to make Collections of projects, components, and registrations
+    for whatever purpose the user might want to organize them. They make node_links to any Node that a user has
+    read access to. Collections through this API do not nest. Currently Collections are private to any individual user,
+    though that could change one day.
+
+    ##Collection Attributes
+
+    OSF Organizer Collection entities have the "nodes" `type`.
+
+        name           type               description
+        ---------------------------------------------------------------------------------
+        title          string             title of Organizer Collection
+        date_created   iso8601 timestamp  timestamp that the collection was created
+        date_modified  iso8601 timestamp  timestamp when the collection was last updated
+
+
+    ##Links
+
+    See the [JSON-API spec regarding pagination](http://jsonapi.org/format/1.0/#fetching-pagination).
+
+    ##Actions
+
+    ###Creating New Organizer Collections
+
+        Method:        POST
+        URL:           links.self
+        Query Params:  <none>
+        Body (JSON):   {
+                         "data": {
+                           "type": "collections", # required
+                           "attributes": {
+                             "title":       {title},          # required
+                           }
+                         }
+                        }
+        Success:       201 CREATED + collection representation
+
+    New Organizer Collections are created by issuing a POST request to this endpoint.  The `title` field is
+    mandatory. All other fields not listed above will be ignored.  If the Organizer Collection creation is successful
+    the API will return a 201 response with the representation of the new node in the body.
+    For the new Collection's canonical URL, see the `links.self` field of the response.
+
+    ##Query Params
+
+    + `page=<Int>` -- page number of results to view, default 1
+
+    + `filter[<fieldname>]=<Str>` -- fields and values to filter the search results on.
+
+    Organizer Collections may be filtered by their `title`, which is a string field and will be filtered using simple
+    substring matching.
+
+    #This Request/Response
+
+    """
+    permission_classes = (
+        drf_permissions.IsAuthenticatedOrReadOnly,
+        base_permissions.TokenHasScope,
+    )
+
+    required_read_scopes = [CoreScopes.ORGANIZER_COLLECTIONS_BASE_READ]
```

Comment:
True. Although, in this case, the node_links view uses the nodes metadata, and the collections view just does collections.